### PR TITLE
Adding concept combinators to NoPCM, SWHS

### PR DESCRIPTION
**Pull request by Ant13731**
Since these two examples mainly use SWHS files, I just decided to group them together for this PR.
Contributes #2499

### CODE REVIEW

File: code/drasil-example/Drasil/NoPCM/Requirements.hs [hunk 1]
```diff
@@ -39,8 +40,8 @@ findMass = findMassConstruct inputInitVals (phrase mass) [eBalanceOnWtr]
 --
 oIDQVals :: [Sentence]
 oIDQVals = map foldlSent_ [
-  [S "the", plural value, S "from", makeRef2S inputInitVals],
-  [S "the", phrase mass, S "from", makeRef2S findMass],
+  [pluralNP (the value), S "from", makeRef2S inputInitVals],
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/NoPCM/Requirements.hs:43]: I'm pretty sure there is a combinator for adding "from" and a reference.

File: code/drasil-example/Drasil/SWHS/Requirements.hs [hunk 2]
```diff
@@ -66,22 +68,22 @@ findMassConstruct fr m ims ddefs = cic "findMass" (foldlSent [
...
-  titleize output_, S "the", plural inValue `S.and_`
+  titleize output_, pluralNP (the inValue) `S.and_`
   S "derived", plural value `S.inThe` S "following list"] +:+.
   foldlList Comma List x) "Output-Input-Derived-Values" funcReqDom
 
 oIDQVals :: [Sentence]
 oIDQVals = map foldlSent_ [
-  [S "the", plural value, S "from", makeRef2S (inReq EmptyS)],
-  [S "the", plural mass, S "from", makeRef2S findMass],
+  [pluralNP (the value), S "from", makeRef2S (inReq EmptyS)],
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/SWHS/Requirements.hs:85]: Here as well.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]
Also, minor thing: it is better to separate infrastructure PRs (i.e. adding new combinators) from PRs that upgrade examples on a variety of fronts.

File: code/drasil-example/Drasil/NoPCM/Requirements.hs [hunk 3]
```diff
@@ -39,9 +40,9 @@ findMass = findMassConstruct inputInitVals (phrase mass) [eBalanceOnWtr]
 --
 oIDQVals :: [Sentence]
 oIDQVals = map foldlSent_ [
-  [S "the", plural value, S "from", makeRef2S inputInitVals],
-  [S "the", phrase mass, S "from", makeRef2S findMass],
-  [ch balanceDecayRate, sParen (S "from" +:+ makeRef2S balanceDecayRate)]
+  [pluralNP (the value), fromSource inputInitVals],
+  [phraseNP (the mass), fromSource findMass],
+  [ch balanceDecayRate, fromSource balanceDecayRate]
```
Comment (Ant13731) [inline code/drasil-example/Drasil/NoPCM/Requirements.hs:45]: Using fromSource on `the value` and `the mass` wraps those references in parenthesis (which was not there previously). Should I keep it this way? The `balanceDecayRate` one already had parenthesis previously, so it worked well.

Comment (JacquesCarette) [reply code/drasil-example/Drasil/NoPCM/Requirements.hs:45]: increased uniformity is good, so yes, keep that.

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**